tests/phpunit/elementor/core/kits/documents/kits/test-kit.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Core\Kits\Documents;
+
+use Elementor\Modules\History\Revisions_Manager;
+use Elementor\Plugin;
+use Elementor\Testing\Elementor_Test_Base;
+
+class Test_Kit extends Elementor_Test_Base {
+
+	/**
+	 * @var \Elementor\Core\Kits\Documents\Kit
+	 */
+	private $kit;
+
+	public function setUp() {
+		wp_set_current_user( $this->factory()->get_administrator_user()->ID );
+
+		$kit = Plugin::$instance->kits_manager->get_active_kit();
+		$this->kit = Plugin::$instance->documents->get( $kit->get_id(), false );
+
+		// In the production environment 'JS' sends empty array, do the same.
+		add_post_meta( $kit->get_main_id(), '_elementor_data', '[]' );
+	}
+
+	public function test_save_kits_revision_ensure_changed() {
```

Comment:
Separate the method name from the subject with `__`.
```suggestion
	public function test_save___kits_revision_ensure_changed() {
```

Reply:
Handled.